2024/10.md

```diff
@@ -126,7 +126,7 @@ When applicable, use these emoji as a prefix to the agenda item topic.
 
     | timebox | topic | presenter |
     |:-------:|-------|-----------|
-    | 45m | WHATWG Observables | Dominic Farolino |
+    | 45m | 🔒 WHATWG Observables | Dominic Farolino |
```

Comment:
Daniel Minor's entry (also subject to scheduling constraints) does not include this emoji, but I think if I'm reading the emoji table correctly, this is the right formatting?

Reply:
you're right, both should